### PR TITLE
on Qt changing tooltip text not working

### PR DESCRIPTION
**Pull request by AliKet**

### CODE REVIEW

File: src/qt/tooltip.cpp [hunk 1]
```diff
@@ -59,6 +63,7 @@ const wxString &wxToolTip::GetTip() const
 
 void wxToolTip::SetWindow(wxWindow *win)
 {
+    wxASSERT(win);
```
Comment (vadz) [inline src/qt/tooltip.cpp]: Should be `wxCHECK_RET()` instead.

File: src/qt/window.cpp [hunk 2]
```diff
@@ -951,14 +951,22 @@ void wxWindowQt::DoMoveWindow(int x, int y, int width, int height)
...
 {
+    if ( m_tooltip == tip )
+        return;
+
     wxWindowBase::DoSetToolTip( tip );
 
-    if ( tip != NULL )
-        GetHandle()->setToolTip( wxQtConvertString( tip->GetTip() ));
+    if ( m_tooltip )
+        m_tooltip->SetWindow(static_cast<wxWindow*>(this));
```
Comment (vadz) [inline src/qt/window.cpp]: Cast seems unnecessary?

**Review by vadz — COMMENTED** [review]
Looks good, thanks, just a couple of minor nitpicks.

File: src/qt/window.cpp (hunk 2 above)
Comment (AliKet) [reply src/qt/window.cpp]: Yes, the case is unnecessary here.
corrected now.

**Comment by vadz** [comment]
Thanks!

**Pull request closed without merging**